src/business.ts

```diff
@@ -0,0 +1,121 @@
+import type { Marqeta, MarqetaOptions, MarqetaError } from './'
+
+export interface BusinessIdentifications {
+  type: string;
+  value: string;
+}
+
+export interface IncorporationType {
+  incorporationType: string;
+  stateOfIncorporation: string;
+}
+
+export interface Address {
+  address1: string;
+  address2: string;
+  city: string;
+  state: string;
+  postalCode: string;
+  country: string;
+}
+
+export interface Personal {
+  firstName: string;
+  lastName: string;
+  home: Address;
+}
+
+export interface Meta {
+  name: string;
+}
+
+export interface Business {
+  token?: string,
+  createdTime: string,
+  metadata: Meta,
```

Comment:
The real type for `metadata` is either `any`, which has no checks on it, or if you really want to have something with types, then you can try:

```typescript
  metadata: { [index: string] : number | string | string[] | boolean },
```

but this only allows for strings for the keys and numbers, strings, string arrays, and booleans for the values - so no nested objects. This is pretty limiting, so I'd go with `any`.